.github/workflows/chromatic-pr.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         fetch-depth: 0
     - uses: actions/setup-node@v1
       with:
-        node-version: '12'
+        node-version: '>=14.16.0 14'
```

Comment:
why we are passing `14.16.0` and `14` as a separate two values?

Reply:
`I've restricted the range to keep it on the current LTS (14.x).`

Reply:
It's saying node version 14.16.0 and above, but you can't go to v15 or above

Reply:
This means any version of 14 greater than 14.16.0 but not 15.

Reply:
Understood :)